### PR TITLE
Jakarta Faces 3.0 Ballot Results

### PR DESCRIPTION
**Pull request by Pandrex247**
Ballot results for Jakarta Faces 3.0.
https://github.com/jakartaee/specifications/pull/278

Signed-off-by: Andrew Pielage <pandrex247@hotmail.com>

### CODE REVIEW

File: faces/3.0/_index.md [hunk 1]
```diff
@@ -37,20 +37,25 @@ Please reference that ballot for the official results.
...
-| Marcelo Ancelmo, Martijn Verburg                    | Participant Members |      |
-| Werner Keil                                         | Committer Members   |      |
-| Scott (Congquan) Wang                               | Enterprise Members  |      |
-|                                                     | Total               |      |
+| Kenji Kazumura                                      | Fujitsu             |   +1   |
+| Dan Bandera, Kevin Sutter                           | IBM                 |   +1   |
+| Ed Bratt, Dmitry Kornilov                           | Oracle              |   +1   |
+| Andrew Pielage, Matt Gill                           | Payara              |   +1   |
+| Scott Stark, Mark Little                            | Red Hat             |   +1   |
+| David Blevins, Jean-Louis Monteiro, Cesar Hernandez | Tomitribe           |   +1   |
```
Comment (kwsutter) [inline faces/3.0/_index.md:49]: Nit... Cesar is not a voting member from Tomitribe, his name should be removed from this list.

File: faces/3.0/_index.md [hunk 2]
```diff
@@ -37,20 +37,25 @@ Please reference that ballot for the official results.
...
+| Werner Keil                                         | Committer Members   |   +1   |
+| Scott (Congquan) Wang                               | Enterprise Members  |   +1   |
+|                                                     | Total               |   10   |
+
+Additional Votes:
+- Ed Burns: +1
+- Paul Nicolucci: +1
+
 
 The ballot was run in the [jakarta.ee-spec mailing list]()
```
Comment (kwsutter) [inline faces/3.0/_index.md:61]: Need to provide a link to your initial post to the mailing list asking for a vote on this spec.

**Comment by kwsutter** [comment]
@Pandrex247, if you can clean up these minor items then we can go ahead and get this merged and out of the way.  Thanks!

**Comment by Pandrex247** [comment]
Thanks for the review Kevin, good eyes.

**Review by kwsutter — APPROVED** [review]
LGTM. Thanks, @Pandrex247!

**Pull request merged**